kf_lib_data_ingest/common/misc.py

```diff
@@ -42,15 +43,18 @@ def read_json(filepath, default=None):
     """
     if (default is not None) and (not os.path.isfile(filepath)):
         return default
-
     with open(filepath, 'r') as json_file:
```

Comment:
Instead of changing the default behavior of the `read_json` and `write_json`, can we add a keyword arg (maybe `jsonpickle=True/False` to both methods that will jsonpickle.decode/encode the dict?

I use these two methods from our lib outside of the ingest pipeline to read/write simple json that doesn't need pickling. Since this is a library as well as an app, other people might do the same so it would be good if we can accommodate

Reply:
> I use these two methods from our lib outside of the ingest pipeline to read/write simple json that doesn't need pickling. 

Are you saying that you actually want write_json to throw an exception on complex types? Because otherwise it shouldn't matter. If you don't include complex data, then nothing special will happen to your output.

Reply:
Ohhh I see, I thought it would still write out the data in a verbose form with pickle stuff in there. I think this is fine then 👍

Reply:
eh, I'll throw in an argument anyway

Reply:
If you're going to add the pick kwarg, then maybe also `**kwargs` to the method sigs of `read_json` and `write_json`  and then pass it into the `json.load/json.dump`. I prob should have added that anyway.

Reply:
hmm... I'll add it to dump but I think not to load. If you need special decoding, it seems like you should just be using the pickle option.